packages/orbit-components/src/List/index.tsx

```diff
@@ -1,76 +1,54 @@
 import * as React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import type * as Common from "../common/types";
-import type { Theme } from "../defaultTheme";
-import defaultTheme from "../defaultTheme";
 import { SIZES, TYPES } from "./consts";
-import getSpacingToken from "../common/getSpacingToken";
-import { getLineHeightToken } from "./ListItem";
-import ListContext from "./ListContext";
-import type { Type, Props } from "./types";
-
-const getSizeToken = ({ theme, size }: { theme: Theme; size?: Common.Size }): string | null => {
-  const sizeTokens = {
-    [SIZES.SMALL]: theme.orbit.fontSizeTextSmall,
-    [SIZES.NORMAL]: theme.orbit.fontSizeTextNormal,
-    [SIZES.LARGE]: theme.orbit.fontSizeTextLarge,
-  };
-
-  if (!size) return null;
-
-  return sizeTokens[size];
+import type { Props } from "./types";
+import SPACINGS_AFTER from "../common/getSpacingToken/consts";
+
+const spaceAfterTokens = {
+  [SPACINGS_AFTER.NONE]: "0",
+  [SPACINGS_AFTER.SMALLEST]: "mb-xxs",
+  [SPACINGS_AFTER.SMALL]: "mb-xs",
+  [SPACINGS_AFTER.NORMAL]: "mb-s",
+  [SPACINGS_AFTER.MEDIUM]: "mb-m",
+  [SPACINGS_AFTER.LARGE]: "mb-l",
+  [SPACINGS_AFTER.LARGEST]: "mb-xl",
 };
 
-const getTypeToken = ({ theme, type }: { theme: Theme; type?: Type }): string | null => {
-  const typeTokens = {
-    [TYPES.PRIMARY]: theme.orbit.colorTextPrimary,
-    [TYPES.SECONDARY]: theme.orbit.colorTextSecondary,
-  };
-
-  if (!type) return null;
-
-  return typeTokens[type];
+const sizeTokens = {
```

Comment:
For organization consistency, we tend to have a `helpers` folder with a `twClasses.ts` file where we store this kind of logic

Reply:
while I appreciate the fact that this is how it is commonly done now, I wholeheartedly disagree with the `helpers` and `twClasses` pattern. it is unnecessary indirection that just makes things hard to find, makes us use redundant imports for one-off usages, makes CI slower and makes DX slower

https://softwareengineering.stackexchange.com/questions/422907/is-putting-general-use-functions-in-a-helpers-file-an-anti-pattern-or-code-sme

I challenge this pattern and propose we co-locate object-class maps directly to components where they're used, for:
- ease of discovery
- CI speed (less files and imports)
- less mental overhead and open files in editors

if there's a  **functional** binding between usages, then I propose using a **focused** file/directory where we store such bindings (usually objects with properties shared across multiple components)

and I propose banning files/folders with names including:
- `helpers`
- `utils`
- `classes`
- `tailwind` / `tw`
- `general`
- `shared`
- `common`

Reply:
Thanks for the proposal Boris :) 

while in general, I agree about CI speed, but I would still prefer to have those classes separated sometimes, I think it's more a personal preference or I do not think it's making the DX significantly worse. In the List example, it's completely ok to use it directly where they used it, but in bigger component, it would lead to a much bigger file and then need to scroll a lot :) 

Also, I doubt that our common folder can be completely banned, but we can reduce the helpers/twClasses pattern where it's possible

Reply:
I can believe about CI speed, but I could argue about DX, which, in the end, can be measured by personal preference, as Viktor mentioned. Especially when we have classes that are reused by multiple components, like the spaceAfter, for example. If one changes, I would prefer not to change it manually everywhere it is being used if I can avoid that.

Just like sometimes one could have the `.styled.ts` with all the Styled Components, we could also have the TW classes on a separate file, so we separate styling from logic / behavior and we avoid files with hundreds of lines of code. Or we can do the complete opposite if we find that beneficial. Personally, I would prefer the code more separated and organized (though we can define a different organization) than an all-in-one file. But that's a decision for the whole team.

Reply:
as I said in

> if there's a **functional** binding between usages, then I propose using a focused file/directory where we store such bindings (usually objects with properties shared across multiple components), with a file name that accurately represents the semantics of its contents

we can for example create a `src/spaceAfter.ts` file (or nest focused utilities under an e.g. `layout` folder) and use this. I have no problem with that

as you Viktor mentioned, if we have a massive object that would pollute the `Component/index.tsx` file, there are two things:
- this object should not exist in the first place (to be solved later)
- we indeed can place *huge* objects outside next to the file, or to a `modificators` folder, with a **focused** name, for example `colors.ts`. point is, people _know_ what this folder/file is for. it is not `common`, `shared`, `classes`, or other ambiguous things 😄 I hope you get my drift

Reply:
Agree about that :) Objects will be removed later, yes

we have `utils/layout`, we can break this file into smaller ones, like: 
```
-- utils/layout
-- -- getSpacingToken.ts
-- -- someOther.ts
```

> _**we can for example create a src/spaceAfter.ts file (or nest focused utilities under an e.g. layout folder) and use this. I have no problem with that**_

I think you do not mind if we can keep the `utils` folder if all of them will be with a focused folder and name, right? :D

I will make up some new structure that will follow your suggestion (although it will breaking, as I know for sure that people using our common folder, for instance)

Reply:
we can move this discussion elsewhere, for now I cleaned up my implementation for the scope of this MR